client/src/components/Panel.js

```diff
@@ -42,7 +43,30 @@ const PanelSource = ({links}) => {
   );
 };
 
-export const Panel = ({allowOverflow, title, otherHeaderContent, children, sources, footnotes }) => (
+const PanelGlossary = ({keys}) => {
+  if(_.isEmpty(keys)){
+    return null;
+  }
```

Comment:
I suppose this one can also be written inside `return` with ternary operator as well? Although this is still clean

Reply:
It could, but this code matches the other function (the source links) and I like it this way 😝